src/app.js

```diff
@@ -1,36 +1,64 @@
-const express = require('express');
-const { Cat } = require('./models')
+const express = require("express");
+const { Cat } = require("./models");
 
 const app = express();
 
 app.use(express.json());
 
-/*app.post('/',(req,res) =>  {
-
+app.post("/cats", (req, res) => {
+  Cat.create(req.body)
+    .then((cat) => res.status(201).json(cat))
+    .catch((err) => {
+      console.error("error", err);
+      res.status(400).json({ error: "Bad Request" });
+    });
+});
 
-    Cat.create(req.body).then( cat => res.status(201).json(cat))
-    
-})*/
+app.get("/cats", (req, res) => {
+  Cat.findAll({ where: req.query })
+    .then((cat) => res.status(201).json(cat))
+    .catch((err) => {
+      console.error("error", err);
+      res.status(400).json({ error: "Bad Request" });
+    });
+});
 
-app.post("/cats", (req, res) => {
-    Cat.create(req.body)
-        .then(cat => res.status(201).json(cat))
-        .catch((status, err) => {
-            console.log('err');
-            console.log(err);
-        });
+app.get("/cats/:catId", (req, res) => {
+  Cat.findByPk(req.params.catId)
+    .then((cat) => res.status(201).json(cat))
+    .catch((err) => {
+      console.error("error", err);
+      res.status(400).json({ error: "Bad Request" });
+    });
```

Comment:
It would be good to include a 404 response if the cat id requested didn't exist. At the moment it returns a 201 status with a null response if the entry doesn't exist. There is similar behaviour on the delete and patch requests so it would be good to try to catch these and send 404 responses.